db/schema.rb

```diff
@@ -1283,7 +1283,6 @@
     t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true
     t.index ["twitter_username"], name: "index_users_on_twitter_username", unique: true
     t.index ["username"], name: "index_users_on_username", unique: true
-    t.check_constraint "username IS NOT NULL", name: "users_username_not_null"
```

Comment:
Minor: Possibly this is just GitHub being weird, but this seems like an unrelated change?

Reply:
yeah i just checked main and this change isn’t over there. @jaw6 maybe we could merge this and then do an immediate fast follow to replace the removed line? 

I feel like the validations on the user model itself (ll. 154-154) would be a sturdy enough guard in the meantime.